x/data/server/features/query_attestors.feature

```diff
@@ -0,0 +1,206 @@
+Feature: QueryAttestors
+
+  Background:
+    Given the content entry
+    """
+    {
+      "iri": "regen:13toVfvC2YxrrfSXWB5h2BGHiXZURsKxWUz72uDRDSPMCrYPguGUXSC.rdf",
+      "timestamp": "2020-01-01T00:00:00.000000000Z"
+    }
+    """
+
+  Rule: one attestor is returned if one attestor entry exists
```

Comment:
I don't think this is an accurate use of the Rule keyword but it would be nice to group cases together that use the same Background. Should this just be a scenario with sub-scenarios?

Reply:
why can't this all be in the main background? or does it make sense to have separate files? i agree this isn't really a "rule"

Reply:
> why can't this all be in the main background?

The attestor entries were not in the main background because there is a scenario for each query in which no attestor entries exist, but it might make more sense to create a separate content entry in the main background.

We also might want to consider avoiding the Background keyword. The authors of the BDD books have requested the Background keyword be removed from Gherkin and they say it may be removed in the future (with options to support backwards compatibility). "We believe that the Background keyword should not be used, because it has a negative effect on the readability of the scenarios."

> or does it make sense to have separate files?

I was mimicking the current file structure and attempting to limit duplication, but I think these are technically separate features so separate files would probably make more sense. Also, the BDD Formulation book recommends simplicity and readability over duplication and redundancy when writing test code.

Reply:
> We also might want to consider avoiding the Background keyword.

This might be more relevant when illustrating user stories and less so in cases like this.